PluralKit.Bot/Commands/Groups.cs

```diff
@@ -577,6 +577,11 @@ public async Task DeleteGroup(Context ctx, PKGroup target)
         await ctx.Reply($"{Emojis.Success} Group deleted.");
     }
 
+    public async Task DisplayId(Context ctx, PKGroup target)
+    {
+        await ctx.Reply($"{target.Hid}");
```

Comment:
it doesn't really make sense to use string formatting if you're just going to put a single string in it
either way, this probably should be in a codeblock, similar to other plaintext output

Reply:
These are specifically not in a codeblock because the mobile clients will copy the backticks - which partially defeats the purpose of having these commands (to make it easier to copy IDs on mobile)

Reply:
oh :/